src/modules/common/components/__tests__/FailureSwitcher.spec.js

```diff
@@ -29,16 +28,30 @@ describe('FailureSwitcher', () => {
       const error = new ContentNotFoundError({ type: 'event', id: '1234', language, city })
       expect(FailureSwitcher.renderContentNotFoundComponent(error)).toMatchSnapshot()
     })
+
+    it('should render a poi not found failure and match snapshot', () => {
+      const error = new ContentNotFoundError({ type: 'poi', id: 'poiId', language, city })
+      expect(FailureSwitcher.renderContentNotFoundComponent(error)).toMatchSnapshot()
+    })
   })
 
   it('should render a content not found failure if there is a ContentNotFoundError', () => {
-    expect(shallow(
-      <FailureSwitcher error={new ContentNotFoundError({ type: 'extra', id: 'sprungbrett', city, language })}
-                       t={mockTranslate} />
-    )).toMatchSnapshot()
+    const error = new ContentNotFoundError({ type: 'extra', id: 'sprungbrett', city, language })
+    const spy = jest.spyOn(FailureSwitcher, 'renderContentNotFoundComponent')
+
+    const component = mount(<FailureSwitcher error={error} />)
+
+    expect(spy).toHaveBeenCalledWith(error)
+    expect(component.find(Failure)).toHaveLength(1)
+
+    spy.mockRestore()
```

Comment:
Isn't this kind of a duplicate of `should render a extra not found failure and match snapshot`?